src/assets/images/DiscordFilled.svg

```diff
@@ -1,12 +1,10 @@
-    </defs>
+<g clip-path="url(#clip0_2704_43722)">
+<path d="M20.317 4.36973C18.761 3.65549 17.119 3.14617 15.4319 2.85453C15.4166 2.85167 15.4007 2.85372 15.3867 2.86038C15.3726 2.86704 15.3609 2.87797 15.3534 2.89163C15.1424 3.26693 14.9087 3.75643 14.7451 4.14113C12.9004 3.86493 11.0651 3.86493 9.2583 4.14113C9.0947 3.74783 8.8525 3.26693 8.6406 2.89163C8.63276 2.87828 8.62107 2.86762 8.60706 2.86102C8.59306 2.85442 8.57739 2.85219 8.5621 2.85463C6.87488 3.1456 5.2327 3.65487 3.6769 4.36963C3.66359 4.37532 3.65238 4.38499 3.6448 4.39733C0.533404 9.04573 -0.318996 13.5798 0.099204 18.0577C0.100386 18.0687 0.103757 18.0793 0.109116 18.0889C0.114476 18.0986 0.121714 18.107 0.130404 18.1138C2.1832 19.6214 4.1717 20.5366 6.1233 21.1432C6.13848 21.1478 6.15468 21.1475 6.16972 21.1426C6.18477 21.1377 6.19795 21.1283 6.2075 21.1156C6.6691 20.4852 7.0806 19.8204 7.4335 19.1214C7.43835 19.1119 7.44112 19.1014 7.44163 19.0906C7.44213 19.0799 7.44036 19.0692 7.43643 19.0592C7.4325 19.0493 7.4265 19.0402 7.41883 19.0327C7.41115 19.0252 7.40198 19.0194 7.3919 19.0157C6.7391 18.7681 6.1176 18.4662 5.5197 18.1234C5.5088 18.117 5.49964 18.108 5.49304 18.0973C5.48643 18.0865 5.48258 18.0742 5.48183 18.0616C5.48108 18.049 5.48345 18.0364 5.48873 18.0249C5.49401 18.0134 5.50204 18.0034 5.5121 17.9957C5.6379 17.9014 5.7638 17.8034 5.8839 17.7043C5.89458 17.6955 5.9075 17.6899 5.92121 17.688C5.93492 17.6862 5.94887 17.6882 5.9615 17.6938C9.8893 19.4871 14.1415 19.4871 18.0229 17.6938C18.0355 17.6878 18.0496 17.6855 18.0635 17.6872C18.0774 17.6889 18.0906 17.6945 18.1014 17.7033C18.2216 17.8023 18.3474 17.9014 18.4742 17.9957C18.4843 18.0033 18.4924 18.0132 18.4977 18.0247C18.5031 18.0361 18.5055 18.0487 18.5049 18.0613C18.5042 18.0739 18.5005 18.0861 18.494 18.097C18.4875 18.1078 18.4784 18.1168 18.4676 18.1233C17.8695 18.4727 17.2429 18.7709 16.5946 19.0147C16.5845 19.0186 16.5754 19.0245 16.5678 19.0321C16.5602 19.0398 16.5542 19.0489 16.5504 19.059C16.5466 19.069 16.5449 19.0798 16.5455 19.0906C16.5461 19.1013 16.549 19.1119 16.5539 19.1214C16.9143 19.8194 17.3258 20.4842 17.7789 21.1146C17.7882 21.1276 17.8013 21.1374 17.8164 21.1425C17.8315 21.1477 17.8478 21.1479 17.8631 21.1432C19.8241 20.5365 21.8126 19.6213 23.8654 18.1138C23.8742 18.1074 23.8816 18.0991 23.887 18.0896C23.8924 18.0801 23.8957 18.0695 23.8967 18.0586C24.3971 12.8816 23.0585 8.38473 20.3482 4.39823C20.3416 4.38528 20.3305 4.37513 20.317 4.36963V4.36973ZM8.02 15.3311C6.8375 15.3311 5.8631 14.2454 5.8631 12.9121C5.8631 11.5789 6.8186 10.4932 8.0201 10.4932C9.2309 10.4932 10.1958 11.5884 10.1769 12.9122C10.1769 14.2454 9.2214 15.3311 8.02 15.3311V15.3311ZM15.9948 15.3311C14.8123 15.3311 13.8379 14.2454 13.8379 12.9121C13.8379 11.5789 14.7933 10.4932 15.9948 10.4932C17.2056 10.4932 18.1705 11.5884 18.1516 12.9122C18.1516 14.2454 17.2056 15.3311 15.9948 15.3311V15.3311Z" fill="white"/>
+</g>
+<defs>
+<clipPath id="clip0_2704_43722">
+<rect width="24" height="24" fill="white"/>
```

Comment:
we need this to be `current` so that it is flexible and we can specify the color as a `fill` attribute on the icon. This way we don't have to export svg's for every single color we want to support, we can use a design system color, and we can do things like animate it or change the color onHover, etc.

Reply:
They are current now, but even applying the fill in SVG path is not working, it makes the icons invisible.

Reply:
Changed